### ISSUE TITLE
Bugs record

### ISSUE THREAD

**Issue by ruiyan1995** [joaanna/something_else#5]
Thanks for your wonderful work! I record some minor bugs as follows,
1. Could you provide the script for extracting frames? (I just used the script provided in [here](https://github.com/facebookresearch/SlowFast/blob/master/slowfast/datasets/DATASET.md#something-something-v2) with **12 FPS**)
2. line 184, code/train.py: no attribute 'arch', and no such directory './ckpt/xxxxx' (need to create the path of args.ckpt and delete 'arch');
3. line 284, code/data_utils/data_loader_frames.py: unnecessary?
4. line 281, code/data_utils/data_loader_frames.py: the 'global_box_id' will be larger than 'num_boxes', thus needs "try .. except ..."?
5. line 298, code/model/model_lib.py: the 'train()' function of VideoModelCoordLatentNL that causes to bad results need to be deleted.

More minor bugs will be updated here.

Some important bugs:
1. To aovid random test results, should modify `return ret` to `return [ret[-1]]` at [here](https://github.com/joaanna/something_else/blob/b156bd30930c2d9e28f78244a74b89a7cd1e3c7f/code/data_utils/gtransforms.py#L163)
2. `drop_last = True` at [here](https://github.com/joaanna/something_else/blob/b156bd30930c2d9e28f78244a74b89a7cd1e3c7f/code/train.py#L148) is wrong!

**Comment by joaanna** [comment]
Hi Rui, 

thank you for the bugs report! Feel free to send a pull request.

**Comment by Sobeney** [comment]
@ruiyan1995 how do you solve the bug 4?

**Comment by xxxzhi** [comment]
So many problems...

**Comment by turtleman99** [comment]
For 2, and 3, I think is 'yes'.
For 4, based on the code, I believe they only use one box for one `standard_category` as well as hands. Are there such complicated frames?

**Comment by ruiyan1995** [comment]
@turtleman99 I found that there are more than 4 objects in some frames, but the code did not process this case.

**Comment by turtleman99** [comment]
> @turtleman99 I found that there are more than 4 objects in some frames, but the code did not process this case.

Yes, I found too. To be more clear, the code only addresses only one object for each category and the maximun category number is 4 by default. My solution is to use the annotations with high confidence when collecting them.

**Comment by meifish** [comment]
In line 226 of `data_loader_frame.py`, it seems if the model is appearance based model (e.g. I3D), then the frames would not go through the resize (line 218) and the random crop (line 221), which means that only STIN model would have gone through those.

However, in line 231, if the model is graph objects based (e.g., STIN), then the frames is just a dummy variables (which copy the same frame (retrieved from line 215) the `in_duration` many of times). If the frames using by STIN is just a dummy variables, then why resize and random crop were needed?

Thanks for any feedback~

**Comment by mm840511m** [comment]
Hi, everyone. How can I solve the problem: line 184, code/train.py: no attribute 'arch'?
